argocd/apps/templates/metrics-server.yaml

```diff
@@ -9,8 +9,8 @@ spec:
   project: "devops-stack"
   source:
     repoURL: https://github.com/kubernetes-sigs/metrics-server.git
-    path: deploy/1.8+
-    targetRevision: v0.3.7
+    path: charts/metrics-server
+    targetRevision: release-0.6
```

Comment:
I think a tag should be used instead of a branch.

Reply:
OMG. Of course, you’re right